system/Database/MySQLi/Connection.php

```diff
@@ -416,6 +416,16 @@ public function _fieldData(string $table)
 			sscanf($query[$i]->Type, '%[a-z](%d)', $retval[$i]->type, $retval[$i]->max_length
 			);
 
+			preg_match('#[a-z]+\((\'.+\')\)#', $query[$i]->Type, $matches);
```

Comment:
This block needs documentation and additional tests written for it.

Reply:
For enum values (and float) field types, their options are stored in the type property and need to be extracted out. The regular expression grabs everything in the parenthesis, creates an array of properties split with ',' and if it's no empty it creates the options parameter containing the enum options. Are there any tests for the getFieldData method other then in the ForgeTest?

Reply:
I don't believe there are any other tests for that method other than ForgeTest.